sdk/metric/metrictest/exporter.go

```diff
@@ -64,9 +64,12 @@ func NewTestMeterProvider(opts ...Option) (metric.MeterProvider, *Exporter) {
 	return c, exp
 }
 
-// Library is the same as "sdk/instrumentation".Library but there is
+// Deprecated: please use Scope instead.
+type Library = Scope
+
+// Scope is the same as "sdk/instrumentation".Scope but there is
 // a package cycle to use it so it is redeclared here.
-type Library struct {
+type Scope struct {
```

Comment:
I wasn't exactly sure what I should do here but chose to rename this to Scope and create a type alias to Libary as well. WDYT?